src/components/Modal/Modal.js

```diff
@@ -222,6 +223,8 @@ export default class Modal extends Component {
   };
 
   handleClick = (evt) => {
+
```

Comment:
The console.log :)

Reply:
Great catch, I totally forgot that. Thanks :)